.github/workflows/integration-test.yml

```diff
@@ -1,22 +1,23 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-name: Run Integration Tests
+name: Run EKS addon Integration Tests
```

Comment:
nit: mind renaming the file itself? Perhaps `eks-add-on-integ-test` and `eks-beta-add-on-integ-test`?

Reply:
Updated in revision